l20n/migrate/test_util.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf8
+
+import unittest
+
+import l20n.format.ast as FTL
+
+from transforms import CONCAT, COPY, SOURCE
+from util import ftl_resource_to_ast, fold_ftl
+
+
+def get_source(acc, cur):
+    if isinstance(cur, SOURCE):
+        return acc + ((cur.path, cur.key),)
+    return acc
+
+
+class TestMap(unittest.TestCase):
```

Comment:
I'm not quite sure what this tests? Somehow, this seems to be more of a test for ast.py than migrate/util.py?